app/models/alchemy/content/factory.rb

```diff
@@ -19,7 +19,8 @@ def build(element, essence_hash)
         if (description = content_description(element, essence_hash)).blank?
           raise ContentDefinitionError, "No description found in elements.yml for #{essence_hash.inspect} and #{element.inspect}"
         else
-          new(name: description['name'], element_id: element.id)
+          translate = description['translate'] == false ? false : true
+          new(name: description['name'], element_id: element.id, translate: translate)
```

Comment:
Shouldn't there be a db migration somewhere?

Also, I think there should be a default value in the DB (and maybe change the field to be a negative (e.g. skip_translate) so that nil is treated equally with false.  That way, only the presence of the field in the yaml would cause something unusual.  

example:

in migration:

skip_translate (allow null)

in yaml:

translate: false

the line in this file:

new(name: description['name'], element_id: element.id, skip_translate: description['translate']==false)

This way the only reason to skip translation is if it's explicitly declared in the yaml...and the code looks cleaner.  i'm really tired...did that make sense and do you agree?  it's ok if you don't